doc/manual/faq.rst

```diff
@@ -84,6 +84,15 @@ Copy the ``examples`` folder from that path into your home or user directory, an
 
 If you have progressed past this level and would like to see more in-depth code or real-life examples of how other groups have handled running experiments with ARTIQ, see the "Community code" directory on the M-labs `resources page <https://m-labs.hk/experiment-control/resources/>`_.
 
+fix ``failed to connect to moninj`` in the dashboard?
+-----------------------------------------------------
+
+This and other similar messages almost always indicate that your device database lists controllers (for example, ``aqctl_moninj_proxy``) that either haven't been started or aren't reachable at the given host and port. See :ref:`mgmt-ctlmgr`, or navigate to the directory containing your ``device_db.py`` and run: ::
+
+    $ artiq_ctlgmr
+
+to let the controller manager start the necessary controllers automatically.
+
```

Comment:
I'll drop out the front-end changes and rethink them, since they're not strictly related to the manual changes anyway.

Reply:
Resolving the rest of these comments for now since they're all related to changes that aren't in the PR anymore. But I'll keep them in mind for a re-attempt.